systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -286,6 +290,18 @@ void testMirrorMaker2Metrics() {
         assertThat(values.stream().mapToDouble(i -> i).sum(), is((double) 1));
     }
 
+    @Test
+    void testKafkaBridgeMetrics() {
+        kafkaBridgeMetricsData = MetricsUtils.collectKafkaBridgePodMetrics(BRIDGE_CLUSTER);
+        assertThat("Collected KafkaBridge metrics are empty", kafkaBridgeMetricsData.size() > 0);
+        assertThat("Collected KafkaBridge metrics doesn't contains vertx metrics", kafkaBridgeMetricsData.values().toString().contains("vertx"));
```

Comment:
Let's remember that when the new bridge 0.18.0 will be out, the metrics will contain `strimzi_bridge`